src/App.js

```diff
@@ -8,35 +8,59 @@ import Meeting from './component/Meeting/meeting';
 
 //Rick
 import SignUpPage from './component/signUpPage/signUpPage';
+import ProtectedRoute from './component/protectedRoute'
 
 //Yi-Chun
-import SearchPage from './component/SearchPage/SearchPage';
+// import SearchPage from './component/SearchPage/SearchPage';
```

Comment:
lol, wut? no more search page?

Reply:
It wasn't being used anywhere, so I commented it out because it was causing errors in the console

Reply:
We can comment back in once it's being used

Reply:
I think we just need to add 
<Route path="/SearchPage">
          <SearchPage />
        </Route>